client/state/selectors/get-checklist-task-urls.js

```diff
@@ -49,5 +49,8 @@ export default createSelector(
 			woocommerce_setup: getSiteUrl( state, siteId ) + '/wp-admin/admin.php?page=wc-admin',
 		};
 	},
-	( state, siteId ) => [ getPostsForQuery( state, siteId, FIRST_TEN_SITE_POSTS_QUERY ) ]
+	( state, siteId ) => [
+		getPostsForQuery( state, siteId, FIRST_TEN_SITE_POSTS_QUERY ),
+		getFrontPageEditorUrl( state, siteId ),
```

Comment:
Feels like we should also be adding the other calls to `getPageEditorUrl()` too

```suggestion
		getPostsForQuery( state, siteId, FIRST_TEN_SITE_POSTS_QUERY ),
		getFrontPageEditorUrl( state, siteId ),
		getPageEditorUrl( state, siteId, getContactPage( posts ) ),
		getPageEditorUrl( state, siteId, firstPostID ),
```

Reply:
I think those two are related to `posts` and we already put the `posts` into deps, so we might not need them?

Reply:
I thought perhaps there could be a very unlikely edge case where two sites being switched between were very very similar, so even after switching sites the `getFrontPageEditorUrl()` and `getPostsForQuery()` both returned the same thing. That way the selector cache wasn't refreshed after switching sites.

But I was wrong, `getPostsForQuery()` will return a different array for each site.

After more investigation it looks like many of these urls aren't used anywhere so could be tidied up. But that's another PR.